master/buildbot/db/migrate/versions/001_initial.py

```diff
@@ -20,21 +20,22 @@
 from buildbot.db.migrate_utils import test_unicode
 from buildbot.util import json
 from buildbot.util import pickle
+from buildbot.util import sautils
 from twisted.persisted import styles
 
 metadata = sa.MetaData()
 
-last_access = sa.Table('last_access', metadata,
+last_access = sautils.Table('last_access', metadata,
                        sa.Column('who', sa.String(256), nullable=False),
```

Comment:
Yeah, I was hoping to keep the diff minimal but I'll have to fix that.  More vimming :)